src/ncaa_fb_managers.py

```diff
@@ -190,7 +190,7 @@ def _fetch_odds(self, game: Dict) -> None:
 
             odds_data = self.odds_manager.get_odds(
                 sport="football",
-                league="ncaa_fb",
+                league="college-football",
```

Comment:
I kept getting 400 bad requests when requesting odds for college football, it appears the slug/identifier maybe changed and this is the correct league to use.